Adyen/UI/Form/Items/Split/FormSplitItemView.swift

```diff
@@ -9,39 +9,24 @@ import UIKit
 
 /// A view representing a split item.
 internal final class FormSplitItemView: FormItemView<FormSplitItem> {
```

Comment:
`SplitFormItemView` sounds better IMO

Reply:
Everything is called FormBlablaItem. 
I believe those should align 🤔

Reply:
yeah, true, but I prefer the other way tbh